### PR TITLE
#245 feedbacks on admin page

### PR DESCRIPTION
**Pull request by RostyslavKhasanov**

### CODE REVIEW

File: src/main/java/greencity/controller/PlaceCommentController.java [hunk 1]
```diff
@@ -73,4 +76,32 @@ public ResponseEntity getCommentById(@PathVariable Long id) {
...
+    @ApiPageable
+    @GetMapping("comments")
+    public ResponseEntity getAllComments(@ApiIgnore Pageable pageable) {
+        return ResponseEntity.status(HttpStatus.OK)
+            .body(placeCommentService.getAllComments(pageable));
+    }
+
+    /**
+     * Method that delete comment by id.
+     *
```
Comment (MarianMilian) [inline src/main/java/greencity/controller/PlaceCommentController.java]: Please add swager doc

File: src/main/java/greencity/entity/Comment.java [hunk 2]
```diff
@@ -33,10 +33,12 @@
     @ManyToOne
     private Comment parentComment;
 
-    @OneToMany(mappedBy = "parentComment", fetch = FetchType.LAZY, cascade = {CascadeType.PERSIST, CascadeType.MERGE})
+    @OneToMany(mappedBy = "parentComment", fetch = FetchType.LAZY,
+        cascade = {CascadeType.PERSIST, CascadeType.MERGE, CascadeType.ALL})
     private List<Comment> comments = new ArrayList<>();
 
-    @OneToMany(mappedBy = "comment", fetch = FetchType.LAZY, cascade = {CascadeType.PERSIST, CascadeType.MERGE})
+    @OneToMany(mappedBy = "comment", fetch = FetchType.LAZY,
```
Comment (MarianMilian) [inline src/main/java/greencity/entity/Comment.java]: fetch  type lazy is by default and return cascade type

Comment (RostyslavKhasanov) [reply src/main/java/greencity/entity/Comment.java]: Dear Marian, i fix this problem

File: src/main/java/greencity/controller/PlaceCommentController.java (hunk 1 above)
Comment (RostyslavKhasanov) [reply src/main/java/greencity/controller/PlaceCommentController.java]: Done!

File: src/main/java/greencity/service/PlaceCommentService.java [hunk 3]
```diff
@@ -38,4 +40,12 @@
      * @author Marian Milian.
      */
     void deleteById(Long id);
+
+    /**
+     * Method witch return all comments by page.
+     *
+     * @param pageable pageable configuration.
+     * @author Rostyslav Khasanov.
```
Comment (ZakharSkaletskyi) [inline src/main/java/greencity/service/PlaceCommentService.java]: add @return PageableDto

Comment (RostyslavKhasanov) [reply src/main/java/greencity/service/PlaceCommentService.java]: Done!

**Pull request merged**